sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -802,9 +800,10 @@ protected Symbol visitLikePredicate(LikePredicate node, ExpressionAnalysisContex
             }
             Symbol expression = node.getValue().accept(this, context);
             Symbol pattern = node.getPattern().accept(this, context);
+            Symbol ignoreCase = Literal.of(node.getIgnoreCase() ? "true" : "false");
```

Comment:
why create a boolean string literal to later convert it to a boolean? a `Literal.of(node.getIgnoreCase())` boolean can be used directly here

Reply:
yup, I know, it was simpler to get the concept out. I am going to address this.